local-notifications/src/definitions.ts

```diff
@@ -155,6 +155,24 @@ export interface LocalNotificationsPlugin {
    */
   requestPermissions(): Promise<PermissionStatus>;
 
+  /**
+   * Direct user to the application settings screen to configure exact alarms.
+   *
+   * Only available on Android >= S.
```

Comment:
We usually use the version number (12) instead of the version letter (S), except on push notifications plugin, but there we also indicate the SDK number. 
I think the version number is more helpful for users.